Sources/Amplitude/AMPConstants.m

```diff
@@ -69,6 +69,8 @@
 const int kAMPEventUploadMaxBatchSize = 100;
 const int kAMPEventRemoveBatchSize = 20;
 const int kAMPEventUploadPeriodSeconds = 30; // 30 seconds
+const int kAMPIdentifyUploadPeriodSeconds = 30; // 30 seconds
+const int kAMPMinIdentifyUploadPeriodSeconds = 30; // 30 seconds
```

Comment:
Is this a hard-coded minimum value?

Reply:
Yes, I followed same pattern as `kAMPMinTimeBetweenSessionsMillis`.

`const long kAMPMinTimeBetweenSessionsMillis = 5 * 60 * 1000; // 5 minutes`

Reply:
I see, I was using the interval value directly to check based on our notes. That might be a bit more flexible.

Reply:
Thanks, based on convo from call. Will leave this as is.